memory/ndctl.py

```diff
@@ -237,6 +237,37 @@ def test_namespace(self):
         namespaces = self.get_json('-N')
         self.log.info('Created namespace %s', namespaces)
 
+    def test_namespace_modes(self):
+        """
+        Create  different namespace types
+        """
+        failed = False
```

Comment:
this can be handled looking into the length of failed_modes

Reply:
makes sense.